src/components/button/button.js

```diff
@@ -3,16 +3,16 @@ import PropTypes from 'prop-types';
 
 import styles from './button.module.css';
 
-const Button = ({name}) => {
-    if (name) {
-        return (
-            <button className={styles.button}>{name}</button>
-        );
-    }
+const Button = ({name, onClick}) => {
+    return (
+        <button className={styles.button} onClick={onClick}>{name}</button>
+    );
+
 };
 
 Button.propTypes = {
     name: PropTypes.string,
+    onClick: PropTypes.func.isRequired,
```

Comment:
Are you sure you want to make it required? I can imagine a button without onClick

Reply:
yes, otherwise the code will break here onClick={onClick} with an Undefined